### PR TITLE
Adding a package for cardioid.

### PR DESCRIPTION
**Pull request by rblake-llnl**
Using a non-standard build directory so the package can work with the existing testing code and scripts.  

Designed for use with spack diy for build development.

### CODE REVIEW

File: var/spack/repos/builtin/packages/cardioid/package.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+# ----------------------------------------------------------------------------
```
Comment (ax3l) [inline var/spack/repos/builtin/packages/cardioid/package.py]: please remove these FIXME instructions and only leave the license header :)

File: var/spack/repos/builtin/packages/cardioid/package.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import spack.environment as ev
+import os
+
+
+class Cardioid(CMakePackage):
+    """Cardiac simulation suite."""
+
+    git = "https://github.com/LLNL/cardioid.git"
+
+    # FIXME: Add proper versions and checksums here.
```
Comment (ax3l) [inline var/spack/repos/builtin/packages/cardioid/package.py]: please remove these two template comments

Comment (rblake-llnl) [reply var/spack/repos/builtin/packages/cardioid/package.py]: Thanks!  Added.

File: var/spack/repos/builtin/packages/cardioid/package.py (hunk 1 above)
Comment (rblake-llnl) [reply var/spack/repos/builtin/packages/cardioid/package.py]: Done!

File: var/spack/repos/builtin/packages/cardioid/package.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+
+class Cardioid(CMakePackage):
+    """Cardiac simulation suite."""
+
+    homepage = 'https://baasic.llnl.gov/comp-bio/cardioid-code.php'
+    git = "https://github.com/LLNL/cardioid.git"
+    maintainers = ['rblake-llnl']
+
+    version('master', branch='master')
```
Comment (ax3l) [inline var/spack/repos/builtin/packages/cardioid/package.py]: the deveopment version of a package is special-named `develop` in spack: `version('develop', branch='master')` would be the spelling

**Comment by rblake-llnl** [comment]
I think everything is up to code now?

**Pull request merged**